.meta.toml

```diff
@@ -0,0 +1,22 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
+[meta]
+template = "pure-python"
+commit-id = "f25fcf50dbbae2fb6dfaff61399595e47bf11220"
+
+[python]
+with-appveyor = false
+with-pypy = true
+with-legacy-python = true
+with-docs = false
+with-sphinx-doctests = false
+
+[coverage]
+fail-under = 99
+
+[flake8]
+additional-config = [
+    "# F401 imported but unused",
+    "per-file-ignores =",
+    "    src/zc/queue/__init__.py: F401 F403",
```

Comment:
There's a comment explaining what F401 is, so it would be nice to also have a comment explaining F403.